client/unit-tests/symbolic/test_atomics.py

```diff
@@ -0,0 +1,1111 @@
+import datetime
+
+import pytest
+
+from valor.schemas.symbolic.atomics import (
+    Bool,
+    Date,
+    DateTime,
+    Duration,
+    Equatable,
+    Float,
+    Integer,
+    LineString,
+    MultiLineString,
+    MultiPoint,
+    MultiPolygon,
```

Comment:
some of these types aren't really "atomic types" based on my understanding of that term. should we just call this file `valor.schemas.symoblic.types` or `valor.schemas.symbolic_types`?

Reply:
Ill check to see that `types.py` wont conflict with https://docs.python.org/3/library/types.html